.circleci/config.yml

```diff
@@ -415,6 +416,18 @@ workflows:
           requires:
             - expo_client_approve_build
 
+  shell_app_nightly:
+    triggers:
+      - schedule:
+          cron: "20 5 * * 2,4,6"
```

Comment:
Does this 2,4,6 syntax work with CircleCI? They support only a subset of cron syntax.

Reply:
It should -- they say

> Note: Cron step syntax (for example, */1, */20) is not supported. Range elements within comma-separated lists of elements are also not supported.

but no reference to comma-separated lists themselves not being supported.